### DISCUSSION TITLE
Add stream playing from SD Card for long samples

### DISCUSSION THREAD

**Discussion by modifikator80** [bangcorrupt/freetribe#11] [Ideas]
Modern Samplers can play streaming audio from a memory card without loading them into RAM. It would be great to add such a feature in future firmware versions. Today I am limited to 24.7MB RAM, even though I have a 32GB card, but I can not play long audios from it. Thank you!

**Comment by bangcorrupt** [comment]
This is something I'll try to add in Freetribe, once I have drivers for the SD card and DMA.  It's slightly more difficult than just reading from the card, as the card reader is attached to the CPU and the codec is attached to the DSP.  It would probably be very difficult to work such a large feature into a compiled binary, so it probably won't be available in Hacktribe.

**Reply by modifikator80** [reply]
Thank you! I hope and wait Freetribe!
